### PR TITLE
Add missing header for GoCD 20.1

### PR DESCRIPTION
**Pull request by nicolaasuni**

### CODE REVIEW

File: src/main/java/in/ashwanthkumar/gocd/slack/jsonapi/Server.java [hunk 1]
```diff
@@ -48,17 +48,21 @@ JsonElement getUrl(URL url)
...
             request.setRequestProperty("Authorization", bearerToken);
         } else if (isNotEmpty(mRules.getGoLogin()) && isNotEmpty(mRules.getGoPassword())) {
             String userpass = mRules.getGoLogin() + ":" + mRules.getGoPassword();
-            String basicAuth = "Basic "
+            authHeader = "Basic "
                     + DatatypeConverter.printBase64Binary(userpass.getBytes());
-            request.setRequestProperty("Authorization", basicAuth);
+        }
+        if (authHeader != null) {
+            request.setRequestProperty("Authorization", authHeader);
```
Comment (ashwanthkumar) [inline src/main/java/in/ashwanthkumar/gocd/slack/jsonapi/Server.java:61]: It feels odd to extract only one portion of setting the `Authorization` header out while leaving the other part as is. Can we please revert this line and moving the `Accept` header to above the `if-else` block?

**Review by ashwanthkumar — CHANGES_REQUESTED** [review]
Thank you so much for this change @nicolaasuni. I've added a minor comment on this change, if you could fix that I would be happy to merge this in.

File: src/main/java/in/ashwanthkumar/gocd/slack/jsonapi/Server.java (hunk 1 above)
Comment (nicolaasuni) [reply src/main/java/in/ashwanthkumar/gocd/slack/jsonapi/Server.java:61]: You are right, I didn't pushed the right version.

**Comment by nicolaasuni** [comment]
@ashwanthkumar I have cleaned up the code and added another fix for https://github.com/ashwanthkumar/gocd-slack-build-notifier/issues/105

**Comment by jefftrimm** [comment]
Just a ping to get this PR reviewed and merged as soon as is practical and cut a build.  Everyone running the latest version of GoCD is running into this issue...

**Comment by ashwanthkumar** [comment]
Thanks @nicolaasuni

**Comment by jefftrimm** [comment]
Can you cut a release build with this fix?  Thanks!

**Comment by ashwanthkumar** [comment]
https://github.com/ashwanthkumar/gocd-slack-build-notifier/releases/tag/v2.0.0

**Pull request merged**